.pre-commit-config.yaml

```diff
@@ -14,35 +14,23 @@
 # limitations under the License.
 #
 repos:
-    - repo: https://github.com/pre-commit/pre-commit-hooks
-      rev: v4.0.1
-      hooks:
-          - id: check-merge-conflict
-          - id: debug-statements
-    - repo: https://github.com/psf/black
-      rev: 21.9b0
-      hooks:
-          - id: black
-    - repo: https://github.com/PyCQA/isort
-      rev: 5.9.3
-      hooks:
-          - id: isort
-            args: ["--profile", "black"]
-    # -   repo: https://github.com/pre-commit/mirrors-mypy
-    #     rev: v0.910-1
-    #     hooks:
-    #     -   id: mypy
-    #         additional_dependencies: [types-all]
-    # - repo: https://github.com/PyCQA/flake8
-    #   rev: 4.0.1
-    #   hooks:
-    #       - id: flake8
-    #         additional_dependencies:
-    #         - flake8-black>=0.1.1
-    #         language_version: python3          
-    - repo: https://github.com/asottile/pyupgrade
-      rev: v2.29.1
-      hooks:
-          - id: pyupgrade
-            args: [--py38-plus]
-
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+    -   id: check-merge-conflict
+    -   id: debug-statements
+-   repo: https://github.com/asottile/pyupgrade
+    rev: v2.29.0
+    hooks:
+    -   id: pyupgrade
+        args:
+        - --py3-plus
+-   repo: https://github.com/psf/black
+    rev: 21.12b0
+    hooks:
+    -   id: black
+-   repo: https://github.com/PyCQA/isort
+    rev: 5.10.1
+    hooks:
+    -   id: isort
+        args: ["--profile", "black"]
```

Comment:
You can run `pre-commit autoupdate` to get the latest versions.

Reply:
Done